File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/OcsEmr/OCS2015U06EmrTemplateInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.OcsEmr
{
    [Serializable]
	public class OCS2015U06EmrTemplateInfo
	{
		private String _templateId;
		private String _templateTypeId;
		private String _templateCode;
		private String _templateName;
		private String _templateContent;
		private String _description;
		private String _gwa;
		private String _userId;
		private String _permissionType;
		private String _created;
		private String _updated;

		public String TemplateId
		{
			get { return this._templateId; }
			set { this._templateId = value; }
		}

		public String TemplateTypeId
		{
			get { return this._templateTypeId; }
			set { this._templateTypeId = value; }
		}

		public String TemplateCode
		{
			get { return this._templateCode; }
			set { this._templateCode = value; }
		}

		public String TemplateName
		{
			get { return this._templateName; }
			set { this._templateName = value; }
		}

		public String TemplateContent
		{
			get { return this._templateContent; }
			set { this._templateContent = value; }
		}

		public String Description
		{
			get { return this._description; }
			set { this._description = value; }
		}

		public String Gwa
		{
			get { return this._gwa; }
			set { this._gwa = value; }
		}

		public String UserId
		{
			get { return this._userId; }
			set { this._userId = value; }
		}

		public String PermissionType
		{
			get { return this._permissionType; }
			set { this._permissionType = value; }
		}

		public String Created
		{
			get { return this._created; }
			set { this._created = value; }
		}

		public String Updated
		{
			get { return this._updated; }
			set { this._updated = value; }
		}

		public OCS2015U06EmrTemplateInfo() { }

		public OCS2015U06EmrTemplateInfo(String templateId, String templateTypeId, String templateCode, String templateName, String templateContent, String description, String gwa, String userId, String permissionType, String created, String updated)
		{
			this._templateId = templateId;
			this._templateTypeId = templateTypeId;
			this._templateCode = templateCode;
			this._templateName = templateName;
			this._templateContent = templateContent;
			this._description = description;
			this._gwa = gwa;
			this._userId = userId;
			this._permissionType = permissionType;
			this._created = created;
			this._updated = updated;
		}

	}
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/emr/OCS2015U06EmrTemplateInfo.java
package nta.med.data.model.ihis.emr;

import java.math.BigDecimal;
import java.util.Date;

public class OCS2015U06EmrTemplateInfo {
    private Integer templateId;
    private Integer templateTypeId;
    private String templateCode;
    private String templateName;
    private String templateContent;
    private String hospCode;
    private String gwa;
    private String userId; 
    private String permissionType;
    private BigDecimal activeFlg;
    private Date created; 
    private Date updated;
    
	public OCS2015U06EmrTemplateInfo(Integer templateId,
			Integer templateTypeId, String templateCode, String templateName,
			String templateContent, String hospCode, String gwa, String userId,
			String permissionType, BigDecimal activeFlg, Date created,
			Date updated) {
		super();
		this.templateId = templateId;
		this.templateTypeId = templateTypeId;
		this.templateCode = templateCode;
		this.templateName = templateName;
		this.templateContent = templateContent;
		this.hospCode = hospCode;
		this.gwa = gwa;
		this.userId = userId;
		this.permissionType = permissionType;
		this.activeFlg = activeFlg;
		this.created = created;
		this.updated = updated;
	}
	public Integer getTemplateId() {
		return templateId;
	}
	public void setTemplateId(Integer templateId) {
		this.templateId = templateId;
	}
	public Integer getTemplateTypeId() {
		return templateTypeId;
	}
	public void setTemplateTypeId(Integer templateTypeId) {
		this.templateTypeId = templateTypeId;
	}
	public String getTemplateCode() {
		return templateCode;
	}
	public void setTemplateCode(String templateCode) {
		this.templateCode = templateCode;
	}
	public String getTemplateName() {
		return templateName;
	}
	public void setTemplateName(String templateName) {
		this.templateName = templateName;
	}
	public String getTemplateContent() {
		return templateContent;
	}
	public void setTemplateContent(String templateContent) {
		this.templateContent = templateContent;
	}
	public String getHospCode() {
		return hospCode;
	}
	public void setHospCode(String hospCode) {
		this.hospCode = hospCode;
	}
	public String getGwa() {
		return gwa;
	}
	public void setGwa(String gwa) {
		this.gwa = gwa;
	}
	public String getUserId() {
		return userId;
	}
	public void setUserId(String userId) {
		this.userId = userId;
	}
	public String getPermissionType() {
		return permissionType;
	}
	public void setPermissionType(String permissionType) {
		this.permissionType = permissionType;
	}
	public BigDecimal getActiveFlg() {
		return activeFlg;
	}
	public void setActiveFlg(BigDecimal activeFlg) {
		this.activeFlg = activeFlg;
	}
	public Date getCreated() {
		return created;
	}
	public void setCreated(Date created) {
		this.created = created;
	}
	public Date getUpdated() {
		return updated;
	}
	public void setUpdated(Date updated) {
		this.updated = updated;
	}
    
}
